qiskit_experiments/library/randomized_benchmarking/interleaved_rb_analysis.py

```diff
@@ -57,44 +56,41 @@ class InterleavedRBAnalysis(RBAnalysis):
 
         See Ref. [1] for more details.
 
+    # section: fit_model
+        Fit model for standard RB
 
+        .. math::
+            F(x) = a \alpha^{x} + b
 
-    Fit Model
-        The fit is based on the following decay functions:
+        Fit model for interleaved RB
```

Comment:
I think it's good if the text is written in a way that's easy to read fluently, and with some redundancy. Why not draw the attention to the fact that the fit function is a decay.

Reply:
Actually original formatting is broken (sentence is injected in math mode) but I can add description of curve like this.